mvc/views/view-components.rst

```diff
@@ -21,54 +30,98 @@ New to ASP.NET MVC 6, view components are similar to partial views, but they are
  
 One use of a view component could be to create a login panel that would be displayed on every page with the following functionality:
 
-- If the user is not logged in, a login panel is rendered.
-- If the user is logged in, links to log out and manage account are rendered.
-- If the user is in the admin role, an admin panel is rendered.
+- If the user is not logged in, a login panel is rendered
+- If the user is logged in, links to log out and manage account are rendered
+- If the user is in the admin role, an admin panel is rendered
+
+The view component ``User`` property gives you access to a user's claims, so you can render data depending on the user's claims. You can add this view component view to the layout page and have it render user-specific data throughout the whole application. View components don’t use model binding, and only depend on the data you provide when calling into it. 
+
+A view component consists of two parts, the class (typically derived from  ``ViewComponent``) and the Razor view which calls methods in the view component class). Like controllers, a view component can be a POCO, but most developers will want to take advantage of the methods and properties available by deriving from ``ViewComponent``.
 
-You can also create a view component that gets and renders data depending on the user's claims. You can add this view component view to the layout page and have it get and render user-specific data throughout the whole application. View components don’t use model binding, and only depend on the data you provide when calling into it. 
+See `ViewComponent properties <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/ViewComponent/index.html#properties>`__ and `ViewComponent Class <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/ViewComponent/index.html>`__.
```

Comment:
Instead of explicitly pointing to the API ref docs with this extra sentence just make the previous mention of the ViewComponent type a link to the ViewCompoent class API ref topic.